pulp_ansible/tests/functional/utils.py

```diff
@@ -322,7 +322,11 @@ def _create_empty_repo_and_distribution(self, cleanup=True):
 
 
 settings = Dynaconf(
-    settings_files=["pulp_ansible/tests/assets/func_test_settings.py", "/etc/pulp/settings.py"]
+    settings_files=[
+        "pulp_ansible/tests/assets/func_test_settings.py",
```

Comment:
the tests couldn't get this, so it didn't load DATABASE setting:
https://github.com/pulp/pulp_ansible/runs/7172035382?check_suite_focus=true#step:11:1493